syncthing_gtk/daemon.py

```diff
@@ -1440,7 +1440,7 @@ def __init__(self):
 
 if __name__ == "__main__":
 	# Small thing for testing
-	from tools import init_logging, set_logging_level
+	from .tools import init_logging, set_logging_level
```

Comment:
Python 3 requires relative imports to be made explicit, so that you can easily distinguish them from external packages. 

Syntax looks like this:

```
from . import tools  # if you want to import the whole module
from .tools import random_function_name, RANDOM_VARIABLE  # if you want to import specific things from the module
```

Reply:
This could also just be changed to

```from syncthing_gtk.tools import init_logging, set_logging_level```

But it seemed like you were consistently importing it this way so I left it alone.  But I don't mind changing it if you would prefer that.